experimental/CliffordAlgebras/src/CliffordAlgebras.jl

```diff
@@ -0,0 +1,579 @@
+export CliffordAlgebra,
```

Comment:
IMO this could be put into one single experimental project. This way, we would not need to care about the correct inclusion order of the two experimental folders (which is an issue atm)